sdk/monitor/monitor-ingestion/src/utils/splitDataToChunksHelper.ts

```diff
@@ -17,7 +17,8 @@ export function splitDataToChunks(logs: Record<string, unknown>[]): Record<strin
       chunk.push(element);
       size += elementSize;
     } else {
-      chunkArray.push(chunk);
+      if(chunk.length)
+        chunkArray.push(chunk);
```

Comment:
Added this check, otherwise there was an edge case, where if the size of a chunk > 1MB then it would have sent an empty chunk along with that chunk. We don't want to send empty chunks.

Reply:
Good catch! Can we add a test for that edge case too?

Reply:
added unit tests